src/geo/ui/search.js

```diff
@@ -1,31 +1,64 @@
 
+/**
+ *  UI component to place the map in the
+ *  location found by the geocoder.
+ *
+ */
+
 cdb.geo.ui.Search = cdb.core.View.extend({
 
   className: 'cartodb-searchbox',
 
+  _ZOOM_BY_CATEGORY: {
+    'building': 18,
+    'postal-area': 15,
+    'default': 12
+  },
+
   events: {
-    "click input[type='text']":   '_focus',
-    "submit form":                '_submit',
-    "click":                      '_stopPropagation',
-    "dblclick":                   '_stopPropagation',
-    "mousedown":                  '_stopPropagation'
+    "click input[type='text']": '_onFocus',
+    "submit form": '_onSubmit',
+    "click": '_stopPropagation',
+    "dblclick": '_stopPropagation',
+    "mousedown": '_stopPropagation'
   },
 
-  initialize: function() {},
+  options: {
+    searchPin: false,
```

Comment:
this is @saleiva's decision: it should be always enabled by default, both in editor and embeds